### ISSUE TITLE
Property hook set with shorthand syntax shows warnings with showTypeOnHover

### ISSUE THREAD

**Issue by MrMeshok** [SanderRonde/phpstan-vscode#124]
```php
<?php

class Example
{
    public int $success {
        set(int $value) {
            $this->success = max($value, 0);
        }
    }
    public int $error1 {
        set {
            $this->error1 = max($value, 0);
        }
    }
    public int $error2 {
        set(int $value) => max($value, 0);
    }
    public int $error3 {
        set => max($value, 0);
    }
}
```
All this hooks are valid but shorthand variants ($error1, $error2, $error3) show warnings 
`PHP Warning:  Undefined array key -1 in ~/.vscode/extensions/sanderronde.phpstan-vscode-4.0.2/_config/be894f2d77ed314152c471e4023e032f/TreeFetcher.php on line 105`

**Comment by SanderRonde** [comment]
Will look into this. In the meantime you can disable the use of this treefetcher by disabling the language server setting (if it's causing you any problems)

**Comment by MrMeshok** [comment]
For now I added `??` on line 150 so it doesn't raise a warning
`$lineStart = $lineOffsets[$line] ?? 0;`
And I still get types from PHPStan on hover
